paddle/fluid/operators/ngraph/ngraph_engine.cc

```diff
@@ -644,8 +650,7 @@ void NgraphEngine::Run(const framework::Scope& scope,
     }
   }
 
-  auto handle = backend_->compile(ng_func);
-  handle->call_with_validate(t_out, *p_t_in);
+  ng_handle->call(t_out, *p_t_in);
```

Comment:
in this way, it can really improve performance.

Reply:
yes, it is faster